src/global/RuntimeParameters.h

```diff
@@ -83,6 +83,16 @@ inline auto& RuntimeParameters() {
         // false,
         // the result will be `NaN` or `infinity` respectively.
         Bool<"division-by-zero-is-undef">{true},
+        // If set to `false`, the contained `FILTER` expressions in the query
+        // try to set and apply a corresponding `PrefilterExpression` (see
+        // `PrefilterExpressionIndex.h`) on its variable-related `IndexScan`
+        // operation.
+        //
+        // If set to `true`, the queries `FILTER` expressions omit setting and
+        // applying `PrefilterExpression`s. This is useful to set a
+        // prefilter-free baseline, or for debugging, as wrong results may be
+        // related to the `PrefilterExpression`s.
+        Bool<"disable-prefilter-for-filter-clauses">{false},
```

Comment:
It should be `enable-prefilter-on-index-scans` 
(no double negation "disabled = false" , as it a) confuses people, and b) is inconsistent with the above parameters.